### DISCUSSION TITLE
How to set up the Project in Ubuntu 20LTS

### DISCUSSION THREAD

**Discussion by ashutosh887** [grafana/grafana#61077] [General]
I'm starting to develop and setup the Project in my local environment but can't initiate it.

<code>make run</code> is stuck at
```
/root/go/bin/wire-v0.5.0 gen -tags "oss" ./pkg/server ./pkg/cmd/grafana-cli/runner
Killed
make[1]: *** [Makefile:77: gen-go] Error 137
make[1]: Leaving directory '/root/grafana'
[Bra] 01-06 05:30:45 [ WARN] Fail to execute command: make [gen-go] - exit status 2
^Cmake: *** [Makefile:106: run] Interrupt
```

**Comment by DrewHorrigan** [comment]
I've had this error today when trying to build Grafana on a virtual machine with 4 Gb of RAM. The problem went away after I increased the RAM to 8 Gb. Presumably, you can instead add a swap file of sufficient size.

**Comment by ashutosh887** [comment]
Thanks a lot @DrewHorrigan

**Comment by gelicia** [comment]
This question looks like it was answered. Closing the discussion.
